llvm-hs/src/LLVM/Internal/FFI/RMWOperation.h

```diff
@@ -0,0 +1,25 @@
+#ifndef __LLVM_INTERNAL_FFI__RMW_OPERATION__H__
+#define __LLVM_INTERNAL_FFI__RMW_OPERATION__H__
+
+/* The AtomicRMWBinOp{FAdd,FSub} were missed from the C-API release of LLVM-9.
+ * Add those manually here. This should be removed in the next release.
+ *
+ * https://github.com/llvm/llvm-project/commit/f57e968dd036b2230c59c00e1ed10fecf1668828#diff-ce8fe77b5ebb1f32a2f9665555fc5f5d
+ */
+typedef enum {
```

Comment:
Can we generate this using `LLVM_HS_FOR_EACH_RMW_OPERATION` so we don’t have to duplicate this list?

Reply:
Good idea, this is done now in the new patch.